collector/boot/src/test/java/org/needlehack/collector/steps/CollectFeedSteps.java

```diff
@@ -6,52 +6,64 @@
 import cucumber.api.java.en.When;
 import org.needlehack.collector.domain.model.feed.Feed;
 import org.needlehack.collector.domain.model.feed.FeedItem;
-import org.needlehack.collector.domain.model.feed.FeedListener;
 import org.needlehack.collector.domain.model.feed.FeedRepository;
 import org.needlehack.collector.helpers.World;
-import org.needlehack.collector.infrastructure.repositories.ElasticFeedItemRepository;
 import org.needlehack.collector.usecases.UseCase;
 import org.needlehack.collector.usecases.rss.params.CollectingFeedParams;
 import org.springframework.beans.factory.annotation.Autowired;
 
 import java.util.List;
 
-public class CollectFeedSteps extends AbstractStepsConfiguration {
-
-    @Autowired
-    private UseCase<CollectingFeedParams> collectFeed;
+import static org.junit.Assert.*;
 
-    @Autowired
-    private FeedRepository feedRepository;
+public class CollectFeedSteps extends AbstractStepsConfiguration {
 
     @Autowired
-    private ElasticFeedItemRepository elasticRepository;
+    private FeedRepository elasticFeedItemRepository;
 
     @Autowired
-    private FeedListener feedListener;
+    private UseCase<CollectingFeedParams> collectFeed;
 
     @Autowired
     private World world;
 
     @Given("^an URI related with an external RSS feed$")
-    public void anURIRelatedWithAnExternalRSSFeed() throws Throwable {
-        world.setFeed(new MockFeed("xml/netflix-techblog.xml", "Netflix"));
+    public void uRIRelatedWithAnExternalRSSFeed() throws Throwable {
+        world.setFeed(new MockFeed("rss/netflix-techblog.xml", "Netflix"));
     }
 
-    @When("^the collecting process is fired up$")
-    public void theCollectingProcessIsFfiredUp() {
-        List<FeedItem> postsCollected = (List<FeedItem>) collectFeed.execute(new CollectingFeedParams(world.getFeed()
+    @Given("^an URI related with an external RSS feed without content$")
+    public void uRIRelatedWithAnExternalRSSFeedWithoutContent() throws Throwable {
+        world.setFeed(new MockFeed("rss/reactnative-techblog.xml", "React Native"));
```

Comment:
Maybe we could extract this RSS URL into an input parameter like as follow:

`@Given("^the URI {string} related with an external RSS feed without content$")
    public void uRIRelatedWithAnExternalRSSFeedWithoutContent(String uri) throws Throwable {
        world.setFeed(new MockFeed(uri, "React Native"));`
   }
`

Reply:
Great ¡¡

I pretend to incorporate this improvement in https://github.com/needlehack-team/needlehack/issues/12